src/controllers/units.controller.js

```diff
@@ -303,8 +303,7 @@ export const update = async (req, res) => {
         // if we are reusing a record, make sure it exists
         await assertRecordExistance(Issuance, updatedRecord.issuance.id);
         updatedRecord.issuanceId = updatedRecord.issuance.id;
-
-        delete updatedRecord.issuance;
+        updatedRecord.issuance.orgUid = orgUid;
```

Comment:
@alexandradumitru22 if you do this then it will be treated as a brand new issuance and not a reused one